ddtrace/contrib/aiobotocore/patch.py

```diff
@@ -56,7 +56,7 @@ async def read(self, *args, **kwargs):
             # inherit parent attributes
             span.resource = self._self_parent_span.resource
             span.span_type = self._self_parent_span.span_type
-            span.meta = dict(self._self_parent_span.meta)
+            span._meta = dict(self._self_parent_span._meta)
```

Comment:
Can we change to not use the internalized meta (with an addition of a `get_tags`)?

```
span.set_tags(self._self_parent_span.get_tags())
```

Reply:
I'm ok with this for now but it should change in the future, there is some other weird stuff happening here with `resource`, `span_type` and `metrics`... Not sure why we're copying them at all.